### DISCUSSION TITLE
Add Average Precision and R2 to the metrics

### DISCUSSION THREAD

**Discussion by thibaultdvx** [Project-MONAI/MONAI#8081] [Ideas]
Hi, 
I would like to suggest to add the [**Average Precision**](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.average_precision_score.html) in the classification metrics. I think that the implementation would be very similar to ROC AUC, so it wouldn't be a big deal to integrate it.
I would be happy to make a PR to do so.

I also feel that an important regression metric is missing:[ **R2 score**](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.r2_score.html). Is there any reason why?
Once again, I would be happy to help to implement it!

WDYT?

**Comment by KumoLiu** [comment]
Hi @thibaultdvx, Welcome, and thank you for your contribution! For more details and guidance, please refer to the [CONTRIBUTION GUIDE](https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md).
